tests/test_url_build.py

```diff
@@ -39,6 +39,9 @@ def test_build_with_port():
     u = URL.build(scheme="http", host="127.0.0.1", port=8000)
     assert str(u) == "http://127.0.0.1:8000"
 
+    u = URL.build(scheme="http", host="127.0.0.1", port="", path="/v1")
+    assert str(u) == "http://127.0.0.1/v1", str(u)
```

Comment:
This and the previous assert are actually the same test. Instead of copying the lines, `@pytest.mark.parametrize` should be used. That previous one shouldn't have been put into the same test as the input validation, anyway.

Could you update this, please?

Reply:
I'll wait for the decision if str is actually acceptable as argument.

Reply:
I think we agreed that it's not acceptable: https://github.com/aio-libs/yarl/pull/959#issuecomment-1821506912.

So that `raises()` check is what actually needs to be parametrized.